docs/ref/models/database-functions.txt

```diff
@@ -340,6 +340,14 @@ returned when this timezone is active will be the same as above except for:
         >>> (dt.isoweekday() % 7) + 1
         2
 
+.. admonition:: ``week`` values
```

Comment:
There's a list above that needs `week` added to it: "Given the datetime 2015-06-15 23:30:01.000321+00:00, the built-in lookup_names return:"